pkg/network/setup/network_test.go

```diff
@@ -99,7 +99,7 @@ var _ = Describe("VMNetworkConfigurator", func() {
 				}}))
 			})
 			It("should accept empty network list", func() {
-				vmi := newVMI("testnamespace", "testVmName")
+				vmi := &v1.VirtualMachineInstance{}
```

Comment:
I'm having trouble matching this change with the content of the commit msg.

Reply:
This is what happens when tests blindly tests something which makes no sense.
If it is not changed to really include no interfaces **and** networks, the test fails.

Changing the production code, cause tests to break and in a chain of reaction to require several fixes.

Summarizing, if production code assumes 1:1 relations between networks and interfaces, tests should also respect that (and `newVMI` was changed to comply with that) and in turn caused this test to fail because it had bad input.

Reply:
AFAIU, having no networks but also no interfaces in the VMI spec **is** correct.

Am I wrong ?..

Reply:
It is correct, but this was not the case. Look again.

Reply:
Yeah, now I realize that `newVMI` did not return what I assumed.